lib/gitlib.sh

```diff
@@ -84,7 +84,8 @@ gitlib::is_repo_admin () {
   else
     logecho $FAILED
     logecho
-    logecho "Authenticated user (GITHUB_TOKEN) must be a repo admin to continue"
+    logecho "You must be a repo admin to continue. Use the 'Request to Join'" \
```

Comment:
maybe change to "You must be a member of the kubernetes-release-managers team"?

Reply:
PTAL